PhysicsTools/NanoAOD/python/muons_cff.py

```diff
@@ -153,6 +181,11 @@
     modifier.toModify(muonTable.variables, puppiIsoId = None, softMva = None)
 
 run2_nanoAOD_102Xv1.toModify(muonTable.variables, puppiIsoId = None)
+print(modifier)
+for modifier in (run2_miniAOD_80XLegacy,run2_nanoAOD_92X,run2_nanoAOD_94X2016,run2_nanoAOD_94XMiniAODv1,run2_nanoAOD_94XMiniAODv2,run2_nanoAOD_102Xv1,run2_nanoAOD_106Xv1,run2_nanoAOD_106Xv2,run3_nanoAOD_122):
```

Comment:
I think the preferred syntax (despite what is used elsewhere in the file) would something like:
```
_modifier = run2_miniAOD_80XLegacy | run2_nanoAOD_92X | ...
_modifier.toModify(...)
```

Reply:
or even better 
`(run2_miniAOD_80XLegacy | run2_nanoAOD_92X | ...).toModify(...).toModify(...)...`